components/email/src/Email.svelte

```diff
@@ -344,6 +344,37 @@
       })
     );
   }
+
+  let currentTrigger, currentTooltip, previousTrigger, previousTooltip;
+
+  function toggleTooltip(e) {
+    currentTrigger = e.target;
+    currentTooltip = currentTrigger.querySelector(".email-tooltip");
+
+    if (
```

Comment:
I am not sure what the intended behaviour here is. But I am assuming that only 1 tooltip within an email component should be visible at a time. If so, this is not always the case.
![Screen Recording 2021-08-09 at 4 50 01 PM](https://user-images.githubusercontent.com/314152/128772869-422214b1-d73f-4e45-b8aa-bfad546227a1.gif)

Reply:
^ don't get that behaviour on FireFox. Might be a browser thing.